website/assets/styles/pages/reports/state-of-device-management.less

```
@@ -118,6 +118,73 @@
       color: @core-vibrant-blue;
       text-decoration: none;
     }
+    [purpose='report-cta-top'] {
+      background-image: url('/images/background-report-cta-top.svg');
+      background-position-x: right;
+      background-size: cover;
+      padding: 60px 40px;
+      border-radius: 16px;
+      margin-bottom: 40px;
+      a {
+        color: #FFF;
+      }
+      [purpose='cta-text'] {
+        color: #FFF;
+        margin-bottom: 24px;
+        h3 {
+          font-weight: 900;
+          font-size: 28px;
+          line-height: 40px;
+          margin-bottom: 4px;
+        }
+        p {
+          font-size: 18px;
+          line-height: 28px;
+          margin-bottom: 0px;
+        }
+      }
+    }
+
+    [purpose='report-cta-bottom'] {
+      background-image: url('/images/background-report-cta-bottom.svg');
+      background-position-x: right;
+      background-size: cover;
+      padding: 70px 40px;
+      border-radius: 16px;
+      margin-bottom: 40px;
```

Please can you add some margin-top to separate it some more from the text above?
```suggestion
      margin-top: 80px;
      margin-bottom: 40px;
```